src/orquestra/sdk/_client/_base/cli/_entry.py

```diff
@@ -426,8 +426,9 @@ def status():
 )
 
 
+# Ignoring pyright errors as this class inherits typing issues form 3rd party code
 # region: login
-class GroupWithDefaultCommand(cloup.Group, DefaultGroup):
+class GroupWithDefaultCommand(cloup.Group, DefaultGroup):  # pyright: ignore
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can we ignore specific errors? not sure if it works
```suggestion
# Ignoring pyright errors as this class inherits typing issues form 3rd party code
# region: login
class GroupWithDefaultCommand(
    cloup.Group, DefaultGroup
):  # pyright: ignore[reportIncompatibleMethodOverride,reportIncompatibleVariableOverride]
```